Apps/Database/src/Models/FileDownload.cs

```diff
@@ -47,13 +47,8 @@ public class FileDownload : AuditableEntity
         /// Gets or sets the id representing the program processing the file.
         /// </summary>
         [Required]
-        public ProgramType ProgramTypeCodeId { get; set; }
-
-        /// <summary>
-        /// Gets or sets the Program Type Code
-        /// Used to define the foreign key in code first.
-        /// </summary>
-        public virtual ProgramTypeCode ProgramTypeCode { get; set; }
+        [MaxLength(10)]
+        public string ProgramCode { get; set; }
```

Comment:
here too

Reply:
This one has docs, not sure what you mean?

Reply:
sorry, I missed the docs from the diff